### DISCUSSION TITLE
LF APAC开源布道者分享系列之九：树立全球观念 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#7] [Announcements]
# LF APAC开源布道者分享系列之九：树立全球观念 - 开源之道

https://opensourceway.community/posts/lfapac-evangelist-sharing/how-to-shape-global-idea-in-os-world/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

其实, 自从 NASA 分享了地球自拍照片后, 全球观念就已经无法从每个人心中清除了,
但是, 知易行难, 有观念和是否有对应行为, 不是一回事儿,
毕竟, 肉身进入另外一个物理空间成本还是太大了...

比如:

> ...任何偏于一隅的项目，拒绝他人参与的项目，都不应该属于开源。

但是, 如果条件所限, 我们只能在中国大局域网中建立开源项目,
又只能在自家主机上发布官网以及代码仓库,
虽然作了一切开源项目必要的开源准备,
网络环境受限, 如果要获得代码, 必须来中国指定城市指定小区内网中进行,
但是, 严格遵守开源许可证, 
慢慢的, 也有外国友人定期来小区内网中同步代码合并 PR,
那么, 这种被迫限制他人参与, 但是, 主观上没有任何拒绝的项目, 是开源嘛?


进一步的, 各种人类学研究都表明, 虽然地球越来越小,
但是, 普通人生活并没因为全球化而得到更多改变,
普通人关注的, 永远是10公里以内的事儿,
就算是通过开源项目接触到全球人士,
最终落到日常生活的, 还是150人以内的强关联人士...

所以, 全球观念, 很可能只是第一动力, 
真正要长期生活其中的, 还是普通的在地反应...这一矛盾是否需要解决?
